config.ts

```diff
@@ -115,7 +115,7 @@ const siteConfig = {
       {
         title: `What is Prisma`,
         description: 'An overview of what Prisma ORM is and how it works.',
-        url: 'concepts/overview/what-is-prisma',
+        url: 'orm/prisma-client/queries/crud',
```

Comment:
Wrong link now.
The CRUD link in line 124 actually also needs to be updated (to this target)

Reply:
doings the lord's work here, Jan! thanks for catching it 🙏